Tasks/ServiceFabricComposeDeploy/ServiceFabricComposeDeploy.ps1

```
@@ -22,25 +22,68 @@ try {
     $removeTimeoutSec = Get-VstsInput -Name removeTimeoutSec
     $getStatusTimeoutSec = Get-VstsInput -Name getStatusTimeoutSec
 
-    $deployParameters = @{
-        'ApplicationName' = $applicationName
-        'Compose' = $composeFilePath
-    }
-    $removeParameters = @{
-        'Force' = $true
-    }
-    $getStatusParameters = @{
-        'ApplicationName' = $applicationName
-    }
-
-    $usePreviewApi = $false
-    $regKey = Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Service Fabric\' -ErrorAction SilentlyContinue
+    $apiVersion = '2.8'
+    $regKey = Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Service Fabric SDK\' -ErrorAction SilentlyContinue
     if ($regKey)
```

should we fail of regkey is not found